src/main/java/org/irods/util/ResourceProperty.java

```diff
@@ -1,5 +1,12 @@
 package org.irods.util;
 
+/**
+ * Enum representing various properties for the iRODS system.
+ * <p>
+ *     This enum is used to ensure that only valid {@code property} values are passed to the modify() method used in
```

Comment:
Is `{@code property}` correct?

Reply:
The enum is for the name of the property to modify

Reply:
I was misunderstanding what `{@code}` was doing.

Carry on.